generic/ltp.py.data/ltp-net.yaml

```diff
@@ -1,22 +1,34 @@
+peer_public_ip: 
+peer_password: 
+peer_user: 
+two_host_configuration : 
 runltp: !mux
     script: 'runltp'
-    net.features:
-        args: '-f net.features'
     net.ipv6:
         args: '-f net.ipv6'
     net.ipv6_lib:
         args: '-f net.ipv6_lib'
-    net.nfs:
-        args: '-f net.nfs'
-    net.tcp_cmds:
-        args: '-f net.tcp_cmds'
     net.sctp:
         args: '-f net.sctp'
     net.multicast:
         args: '-f net.multicast'
-    net.rpc:
-        args: '-f net.rpc'
-    net.rpc_tests:
-        args: '-f net.rpc_tests'
-    net.tirpc_tests:
-        args: '-f net.tirpc_tests'
+    net_stress.appl:
```

Comment:
can we push all net-related test scenarios into another yml file?

Reply:
i see its ltp-net.yaml already

Reply:
ah my bad , thank @abdhaleegit as seeing long list I confused :(